src/Pages/Main/ArticleDiscount/ArticleDiscount.scss

```
@@ -0,0 +1,30 @@
+@import '../../../Style/common.scss';
+
+.articleDiscount {
+  border: 1px solid black;
+  width: 1440px;
+  height: 760px;
+  .articleDiscountBox {
+    width: 1440px;
+    height: 640px;
```

```suggestion

.articleDiscount {
  border: 1px solid black;
  
  .articleDiscountBox {
```
- 새로운 셀렉터 앞에서는 한줄 띄워주세요
- 레이아웃을 잡는 방식에 대해서 다시 생각해보셔야 할 것 같습니다. 레이아웃을 결정하실 때 부모요소에 큰 고정폭은 가급적 사용하지 않습니다. 자식요소의 너비를 누적하여 부모 요소가 결정될 수 있도록 해주세요. 이 방법이 이해가 안되신다면 반드시 재질문해주세요!

---

새로운 셀렉터 마다 한줄 씩 띄웠습니다. 레이아웃을 잡을때 부모부터 시작해서 border 를 줘서 위치를 확인하면서 해가지고 자식요소가 만들어지면 부모요소의 고정폭을 지워줬어야 됐는데 생각하지 못하고 있었습니다. 현재 만들어진 레이아웃의 부모요소에 고정폭을 다 지우겠습니다!